src/Discord/Parts/Channel/Message.php

```diff
@@ -181,6 +181,7 @@ class Message extends Part
     public const FLAG_LOADING = (1 << 7);
     public const FLAG_FAILED_TO_MENTION_SOME_ROLES_IN_THREAD = (1 << 8);
     public const FLAG_SUPPRESS_NOTIFICATIONS = (1 << 12);
+    public const FLAG_V2_COMPONENTS = (1 << 15);
```

Comment:
The official name is `IS_COMPONENTS_V2`

```suggestion
    public const FLAG_IS_V2_COMPONENTS = (1 << 15);
```

please also adjust the referencing code